ln1_f4

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/python
+#coding: utf8
+#Програма повинна запускатися кожних 5хв (12 запусків за год) і записувати статистику використання ресурсів в БД, через годину 
+#дані з БД усереднюються та відправляються на електронку у вигляді таблиці з статистикою
+
+import smtplib
+import subprocess
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+
+data_string = [] #Усі дані зберігаються у форматі cpu_total,cpu_user,cpu_sys,cpu_idle,mem_total,mem_used,mem_free,mem_cached,hdd_total,hdd_used,hdd_free
+
+def get_raw_data(command): #функція виконує консольну команду та повертає результат
+    command_data = subprocess.Popen(command, stdout=subprocess.PIPE, shell=True)
+    (command_data, err) = command_data.communicate()
+    command_data = command_data.replace(",",".") #Заміна ком на крапки для коректного виконання float()
+    return command_data.split()
+
+raw = get_raw_data("mpstat") #обробка даних по завантаженості процесора
+cpu_user = float(raw[21])
+cpu_sys = 0
+for x in raw[22:29]:
+    cpu_sys = cpu_sys + float(x)
+cpu_total = cpu_user + cpu_sys
+cpu_idle = float(raw[30])
+cpu_string = ("CPU - Total used:%s%%, User:%s%%, System:%s%%, Idle:%s%%" % (cpu_total, cpu_user, cpu_sys, cpu_idle))
+print (cpu_string) #в принципі виводити проміжні дані в консоль непотрібно але я залишив це
+
+raw = get_raw_data("free -m") #обробка даних по завантаженості оперативної памяті
+mem_total = int(raw[7])
+mem_used = int(raw[8])
+mem_free = int(raw[9])
+mem_cached = int(raw[12])
+mem_string = ("MEMORY - Total:%sMb, Used:%sMb, Free:%sMb, Cached:%sMb" % (mem_total, mem_used, mem_free, mem_cached))
+print (mem_string)
+
+raw = get_raw_data("df -m --total") #обробка данних по жорсткому диску
+hdd_total = int(raw[50])
+hdd_used = int(raw[51])
+hdd_free = int(raw[52])
+hdd_string = ("Hard disk drive - Total:%sMb, Used:%sMb, Free:%sMb" % (hdd_total, hdd_used, hdd_free))
+print (hdd_string)
+
+                        #формування єдиного рядка з данними для запису в файл
+var_names_list = [cpu_total,cpu_user,cpu_sys,cpu_idle,mem_total,mem_used,mem_free,mem_cached,hdd_total,hdd_used,hdd_free]
```

Comment:
1. the `list` end not needed. I see this is list :)
2. Whitespace's after `,`
3. `computer_resources_info` - match better that `var_names`
4. For now I don't understand why you need two same list of resources, but for copy you can use `new_list = old_list[:]` - it's called slicing.(read about slicing in python). Other copy solutions(http://stackoverflow.com/questions/2612802/how-to-clone-or-copy-a-list-in-python)